index.js

```diff
@@ -16,76 +17,99 @@ function genKey(password, salt) {
     });
   });
 }
-function cryptoInit(password, modP) {
+function cryptoInit(password, options) {
   var db = this;
   var key, pub;
   var turnedOff = false;
-  return db.get(configId).catch(function (err) {
+  var ignore = ['_id', '_rev']
+  var modP
```

Comment:
oops sorry I had it removed, the rebase must have readded it, I’ll fix in in a minute